src/js/store/modules/votes.js

```diff
@@ -95,17 +95,14 @@ const actions = {
 			const response = await axios.get(generateUrl(`${endPoint}/votes`), { params: { time: +new Date() } })
 			const votes = []
 			response.data.votes.forEach((vote) => {
-				if (vote.voteAnswer === 'yes') {
-					vote.voteAnswerTranslated = t('polls', 'yes')
+				if (vote.voteAnswer === 'Yes') {
```

Comment:
Argh. Sorry. The compared keys must stay lower case. Only the translations strings can be changed...

Reply:
> The compared keys must stay lower case.

I was wondering about that too 😁.